templates/identity/email-password/forgot-password-response.html.ejs

```diff
@@ -1,13 +1,20 @@
 <h1>Email sent</h1>
-<form method="post">
-  <p>If your account exists, an email has been sent with a link to reset your password.</p>
-  <p>If you do not receive your email in a couple of minutes, check your spam folder or click the link below to send another email.</p>
+<p>If your account exists, an email has been sent with a link to reset your password.</p>
+<p>If you do not receive your email in a couple of minutes, check your spam folder or try sending another email.</p>
 
-  <input type="hidden" name="email" value="<%= email %>" />
+<ul class="actions">
+  <li><a id="login-link" href="" class="link">Back to Log In</a></li>
+  <li><a id="forgot-link"  href="" class="link">Back to Forgot Password</a></li>
+</ul>
 
-  <p class="actions"><a href="<%= controls.login %>">Back to Log In</a></p>
-
-  <p class="actions">
-    <button type="submit" name="submit" class="link">Send Another Email</button>
-  </p>
-</form>
+<script>
+  // Get controls
+  (async() => {
+    const res = await fetch('<%= idpIndex %>');
+    const json = await res.json();
+    const registerLink = document.getElementById('login-link');
+    registerLink.href = json.controls.login;
+    const forgotLink = document.getElementById('forgot-link');
+    forgotLink.href = json.controls.forgotPassword;
+  })();
```

Comment:
We'll want to extract these scripts into a utility in an external JS file (perhaps in future PR).
I'm thinking

```js
const controls = await loadControls(idpIndex);
controls.setTarget('login-link', registerLink);
controls.setTarget('forgoot-link', forgotLink);
```